admin/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Pronko Consulting (https://www.pronkoconsulting.com)
+ * See LICENSE for the license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="payment">
+            <group id="pronko_liqpay_section" sortOrder="6" showInDefault="1" showInWebsite="1" showInStore="1">
+                <group id="pronko_liqpay" translate="label" type="text" sortOrder="100" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>LiqPay</label>
+                    <fieldset_css>complex</fieldset_css>
+                    <attribute type="displayIn">other_payment_methods</attribute>
+                    <group id="pronko_liqpay_general" translate="label" showInDefault="1" showInWebsite="1" showInStore="1" sortOrder="5">
+                        <label>General Settings</label>
+                        <attribute type="expanded">1</attribute>
+                        <frontend_model>Magento\Config\Block\System\Config\Form\Fieldset</frontend_model>
+                        <field id="active" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                            <label>Enabled</label>
+                            <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                            <config_path>payment/pronko_liqpay/active</config_path>
+                        </field>
+                        <field id="title" translate="label" type="text" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
+                            <label>Title</label>
+                            <config_path>payment/pronko_liqpay/title</config_path>
+                        </field>
+                        <field id="mode" translate="label" type="select" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                            <label>Mode</label>
+                            <source_model>Pronko\LiqPayAdmin\Model\Source\Environment</source_model>
```

Comment:
The source_model has to be renamed to a new class location.

Reply:
@mcspronko Pronko\LiqPayAdmin\Model\Source\Environment is a new class location

Reply:
I thought we will move the class to the Pronko\LiqPayAdmin\Source\Environment

As Model directory does represent only entities which have a database table representation.

Reply:
@mcspronko sorry my mistake. I understood you incorrect. 

Now i fixed it.